exonum-java-binding-core/rust/ejb-app/start_qa_node.sh

```diff
@@ -80,7 +80,7 @@ do
      --ejb-classpath $EJB_CLASSPATH \
      --ejb-libpath $EJB_LIBPATH \
      --ejb-log-config-path $log_config_path \
-     --ejb-debug true \
+     --ejb-jvm-args Xdebug \
```

Comment:
AFAIK, it's not useful by itself (without extra flags), therefore, better be removed.